src/libs/kata-types/src/config/default.rs

```diff
@@ -16,6 +16,8 @@ lazy_static! {
     ];
 }
 pub const DEFAULT_AGENT_NAME: &str = "kata-agent";
+pub const DEFAULT_AGENT_VSOCK_PORT: u32 = 1024;
```

Comment:
`vsock_port` and `log_port` are defined here(`/src/libs/kata-types/src/config/default`).  Is it ok to do this? @jodh-intel